br/br-batch-create-table.md

```diff
@@ -1,18 +1,18 @@
 ---
 title: Batch Create Table
-summary: Learn how to use the Batch Create Table feature. When restoring data, BR can create tables in batches to speed up the restore process.
+summary: Learn how to use the Batch Create Table feature. When restoring data, br command-line tool can create tables in batches to speed up the restore process.
 ---
 
 # Batch Create Table
 
-When restoring data, Backup & Restore (BR) creates databases and tables in the target TiDB cluster and then restores the backup data to the tables. In versions earlier than TiDB v6.0.0, BR uses the [serial execution](#implementation-principles) implementation to create tables in the restore process. However, when BR restores data with a large number of tables (nearly 50000), this implementation takes much time on creating tables.
+When restoring data, br command-line tool creates databases and tables in the target TiDB cluster and then restores the backup data to the tables. In versions earlier than TiDB v6.0.0, the br tool uses the [serial execution](#implementation-principles) implementation to create tables in the restore process. However, when the br tool restores data with a large number of tables (nearly 50000), this implementation takes much time on creating tables.
 
 To speed up the table creation process and reduce the time for restoring data, the Batch Create Table feature is introduced in TiDB v6.0.0. This feature is enabled by default.
 
 > **Note:**
 >
-> - To use the Batch Create Table feature, both TiDB and BR are expected to be of v6.0.0 or later. If either TiDB or BR is earlier than v6.0.0, BR uses the serial execution implementation.
-> - Suppose that you use a cluster management tool (for example, TiUP), and your TiDB and BR are of v6.0.0 or later versions, or your TiDB and BR are upgraded from a version earlier than v6.0.0 to v6.0.0 or later. In this case, BR enables the Batch Create Table feature by default.
+> - To use the Batch Create Table feature, both TiDB and br command-line tool are expected to be of v6.0.0 or later. If either TiDB or br command-line tool is earlier than v6.0.0, the br tool uses the serial execution implementation.
+> - Suppose that you use a cluster management tool (for example, TiUP), and your TiDB and br command-line tool are of v6.0.0 or later versions, or your TiDB and br command-line tool are upgraded from a version earlier than v6.0.0 to v6.0.0 or later. In this case, the br tool enables the Batch Create Table feature by default.
```

Comment:
```suggestion
> - Suppose that you use a cluster management tool (for example, TiUP), and your TiDB and br command-line tool are of v6.0.0 or later versions, or your TiDB and br command-line tool are upgraded from a version earlier than v6.0.0 to v6.0.0 or later. In this case, BR enables the Batch Create Table feature by default.
```

Reply:
This change should synchronize to the Chinese.